tools/expandable/NEURAdemo.py

```diff
@@ -78,24 +78,20 @@ def comp_kernel(self):
         """
         file_source = (self.kernel_name.split("."))[0]
 
-        compile_commands = {
-            (1, 1): f"clang-12 -emit-llvm -fno-unroll-loops -fno-vectorize -O3 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}",
-            (2, 1): f"clang-12 -emit-llvm -funroll-loops -mllvm -unroll-count=2 -fno-vectorize -O3 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}",
-            (4, 1): f"clang-12 -emit-llvm -funroll-loops -mllvm -unroll-count=4 -fno-vectorize -O3 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}",
-            (1, 2): f"clang-12 -emit-llvm -fno-unroll-loops -O3 -mllvm -force-vector-width=2 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}",
-            (1, 4): f"clang-12 -emit-llvm -fno-unroll-loops -O3 -mllvm -force-vector-width=4 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}",
-            (1, 8): f"clang-12 -emit-llvm -fno-unroll-loops -O3 -mllvm -force-vector-width=8 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}"
-        }
-
-        compile_command = compile_commands.get((self.unroll_factor, self.vector_factor))
-        if not compile_command:
+        if self.unroll_factor == 1 and self.vector_factor == 1:
+            compile_command = f"clang-12 -emit-llvm -fno-unroll-loops -fno-vectorize -O3 -o kernel.bc -c ../../test/kernels/{file_source}/{self.kernel_name}"
```

Comment:
Can we make a global constant variable `kKernelDirectory = ../../test/kernels`, and use that across all command/path variables?